driver/src/order_driver.rs

```diff
@@ -94,6 +97,22 @@ fn update_balances(state: &mut State, orders: &[Order], solution: &Solution) {
     }
 }
 
+fn calculate_order_hash(nonreserved_order_hash: H256, standing_orders: Vec<models::StandingOrder>) -> H256 {
```

Comment:
This could also be exposed on the model directly? Technically, you might be able to reuse the `rollingHashable` trait that we support for `Vec<Order>`.